Android.mk

```diff
@@ -91,6 +91,8 @@ SPVTOOLS_OPT_SRC_FILES := \
 		source/opt/types.cpp \
 		source/opt/unify_const_pass.cpp \
 		source/opt/merge_return_pass.cpp
+		sourc/opt/value_number_table.cpp \
+		sourc/opt/value_numbering_pass.cpp
```

Comment:
I renamed this file to local_redundancy_elimination.cpp, but the refactoring tool I used missed this.  This is causing the android build failures.  I'll fix that up.